README.md

```diff
@@ -52,9 +54,15 @@ OGame-clone browser based game engine.
 1. Move source files of the project to your webserver's directory.
 1. Install PHP dependencies.
     - ``composer install --no-dev``
-1. Run installation wizard: http://your_server_address:port/install
+1. Run installation wizard: ``http(s)://<your_server_address>:<port>/install``
 1. Remove ``install/`` directory
 
+### Docker
+
+1. Prepare config file ``mkdir docker && cp config.php docker && cp includes/constants.php docker && chown 33:33 docker/*.php``
```

Comment:
I think it might be beneficial to add a bit of explanation about the `chown 33:33` command here. As far as I understand, it's to make it writeable for the PHP process run inside the container. Also, for some people, it might be necessary to run this command with `sudo`, so maybe it should be added there as well?

Reply:
I have modified the readme, adding a note on this operation. The `chown` command does not require `sudo` for files owned by itself, so there is no need to specify it.

Reply:
This is not always the case. As far as I can tell, your user has to be a member of the target group to be able to change the ownership of a file. In my case, I was not able to do it, since my local user is not part of the `www-data` group on my local machine, leading to an "Operation not permitted" error.

I think it won't hurt to mention this as a note, that for some people, using `sudo chown` might be necessary.

Reply:
OK, I will add `sudo` note in `README.MD`.